docs/ramalama-serve.1.md

```diff
@@ -48,7 +48,7 @@ CONTAINER ID  IMAGE                             COMMAND               CREATED
 
 Generate a quadlet for running the AI Model service
 ```
-$ ramalama serve --generate=quadlet granite
+$ ramalama serve --name MyModel --generate=quadlet granite
```

Comment:
A few questions:

1. Can't the user chose the model? Or is this what `granite` is supposed to mean? If yes, then shouldn't the name fit the one used in the output?
2. Using `MyModel` as the value of the Container name is a bit confusing. I initially thought it was the name of the model